config/pipeline.yaml

```diff
@@ -79,8 +79,25 @@ labs:
 
 test_plans:
 
-  fstests:
-    pattern: 'fstests.jinja2'
+  xfstests-base: &xfstests-base
+    pattern: 'xfstests.jinja2'
+    params: &xfstests-base-params
+      njobs: 3
+      gce: True
```

Comment:
I think this should ideally be handled at the template / runtime level like kunit can be run either in a local shell or in k8s. We should ideally be able to run fstests in a local KVM or GCE with the same test plan name. Maybe it's not practical, but that's the idea with the separation between test plan definitions and runtime environments.

Reply:
That's currently done with `runner.py --lab-config`, except the term "lab" is being abused so the plan is now to replace that with "runtime".  Then a "runtime" can be anything that can generate the template and run it, e.g. a shell, local Docker, k8s, GCE, LAVA etc.  Some test plans might not be compatible with some runtimes, like you can't boot a Raspberry Pi in Kubernetes and you shouldn't run a kernel build on a Raspberry Pi, in which case some missing template layers might make it fail or it won't run when trying to submit it.  But at least the tools would provide a generic mechanism that's agnostic of the type of job and where it's meant to run.  You can already do any combination of kver / kunit tests with shell / k8s runtimes.

Reply:
No that's not what I'm suggesting.  So I don't know how to implement things in this particular case, maybe with a particular runtime for fstests e.g. fstests-gce and when using shell it uses kvm?